test/utils/consoleError.js

```diff
@@ -7,6 +7,12 @@ function consoleError() {
     console.info(...args);
     throw new Error(...args);
   };
+
+  console.warn = (...args) => {
+    // Can't use log as karma is not displaying them.
+    console.info(...args);
+    throw new Error(...args);
```

Comment:
As we have started to use console.warn in the code base, I think that we should throw it for them. I could only find the origin of the new warning thanks to this. Turns out, the table pagination had unmatching values.

cc @eps1lon